src/components/Rgbirdflop/RGBirdflop.tsx

```
@@ -0,0 +1,419 @@
+
+        elementIdToLandOn.value = nextStep.id;
+
+        const nextNotification = new Notification(notification)
+          .setDescription(nextStep.description);
+        notifications.push(nextNotification);
+      }),
+    };
+
+    notifications.push(
+      notification,
+    );
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)